Data/Migrations/20180927022504_AddEventContactEmail.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Data.Migrations
+{
+    public partial class AddEventContactEmail : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {

```

Comment:
The size of this migration suggests you have a problem with your DB config. Your Up/Down should only be affecting the Events table. I would not expect as much as this. Please have @morganbr or @asyasky review this part.